connectors/sources/onedrive.py

```diff
@@ -190,7 +319,7 @@ def get_default_configuration(cls):
                 "label": "Azure application Tenant ID",
                 "order": 3,
                 "type": "str",
-                "value": "tenant#123",
+                "value": "tenant-123",
```

Comment:
This value is used for ftest while calling api

Here the `curl -X POST http://localhost:10972/<tenant-id>/oauth2/v2.0/token` if <tenant-id> is empty string, it will result in error.

Reply:
Update: Found that we can pass defaults to e2e via connector json in fixtures, hence removed these values from here